.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "src/libs/QCBOR"]
 	path = src/libs/QCBOR
 	url = https://github.com/laurencelundblade/QCBOR.git
+[submodule "src/libs/date"]
+	path = src/libs/date
+	url = https://github.com/HowardHinnant/date.git
```

Comment:
I didn't know this was possible. What's the reasoning behind `../../`?
Also, will this work if the project is cloned from another repo (ex : I'm using a private gitea instance as my `origin`) ?

Reply:
the `../../` works just like relative paths on linux filesystems. We go two "folders" up from InfiniTime (so we are at the github "root"), then we go into the "folder" `HowardHinnant/date.git`.

A private gitea instance would work, if (and only if) you have the same structure on your private gitea instance. Otherwise the submodule repos can't be found on the private instance

Maybe in this case the relative path isn't the greatest idea :sweat_smile:

Reply:
The structure of the project (InfiniTime) on my instance is the same than the one on Github. However, the authentification is done on my instance and not on Github, where `HowardHinnant/date.git` is located.